### PR TITLE
Spanish and catalan

### PR DESCRIPTION
**Pull request by AcuarioCat**
Spanish and Catalan

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -17091,7 +17095,7 @@ cz:palmu olejovou
 da:Palmeolie
 de:Palmöl, Ölpalme
 eo:Palmoleo
-es:aceite de palma, grasa de palma, aceite vegetal de palma
+es:aceite de palma, grasa de palma, aceite vegetal de palma,aceite de nuez de palma
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:17098]: aceite de nuez de palma should be in this entry instead:

<en:palm kernel oil and fat
en:palm kernel oil, palm kernel
bs:Ulje palminog jezgra
ca:Oli de palmist
cs:Palmojádrový olej
da:Palmeolie
de:Palmkernöl, Palmkern
es:palmiste

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -33137,7 +33154,7 @@ cy:Llugaeron
 da:Tranebær
 de:Moosbeeren, Cranberry, Cranberrys
 eo:Oksikoko
-es:arándano rojo, arándano
+es:arándano rojo, arándano,arandano,arandanos
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:33157]: arandano,arandanos will not be needed when deaccenting is fixed (soon :) )

Comment (AcuarioCat) [reply taxonomies/ingredients.txt]: ok removed - fingers crossed....

File: taxonomies/ingredients.txt [hunk 3]
```diff
@@ -495,6 +495,9 @@ zh:色素
 wikidata:en:Q753009
 # ingredient/fr:colorant-alimentaire has 67 products in 5 languages @2019-03-27
 
+<en:colour
+en:oncentrated plant colour
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:499]:
```suggestion
en:concentrated plant colour
```

**Pull request merged**